vault/identity_store_oidc.go

```diff
@@ -1481,8 +1491,25 @@ func (i *IdentityStore) pathOIDCIntrospect(ctx context.Context, req *logical.Req
 // namedKey.rotate(overrides) performs a key rotation on a namedKey.
 // verification_ttl can be overridden with an overrideVerificationTTL value >= 0
 func (k *namedKey) rotate(ctx context.Context, logger hclog.Logger, s logical.Storage, overrideVerificationTTL time.Duration) error {
-	verificationTTL := k.VerificationTTL
+	if k.SigningKey == nil {
```

Comment:
Let's do a nil check in the loop through the key ring instead

Reply:
fixed in: https://github.com/hashicorp/vault/pull/13716/commits/7fc38ac71ca846c74943ad2989853b9f431b2397